systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -65,21 +71,20 @@ public class TieredStorageST extends AbstractST {
     private static final String BUILT_IMAGE_TAG = "latest";
     private TestStorage suiteStorage;
 
-    /**
-     * @description This testcase is focused on testing of Tiered Storage integration implemented within Strimzi.
-     * The tests use Aiven Tiered Storage plugin - <a href="https://github.com/Aiven-Open/tiered-storage-for-apache-kafka/tree/main">...</a>
-     *
-     * @steps
-     *  1. - Deploys KafkaNodePool resource with Broker NodePool with PV of size 10Gi
-     *  2. - Deploys Kafka resource with configuration of Tiered Storage for Aiven plugin, pointing to Minio S3, and with image built in beforeAll
-     *  3. - Creates topic with enabled Tiered Storage sync with size of segments set to 10mb (this is needed for speedup the sync)
-     *  4. - Starts continuous producer to send data to Kafka
-     *  5. - Wait until Minio size is not empty (contains data from Kafka)
-     *
-     * @usecase
-     *  - tiered-storage-integration
-     */
     @ParallelTest
+    @TestDoc(
+        description = @Desc("This testcase is focused on testing of Tiered Storage integration implemented within Strimzi. The tests use Aiven Tiered Storage plugin (<a href=\"https://github.com/Aiven-Open/tiered-storage-for-apache-kafka/tree/main\">tiered-storage-for-apache-kafka</a>)."),
+        steps = {
+            @Step(value = "Deploys KafkaNodePool resource with PV of size 10Gi.", expected = "KafkaNodePool resource is deployed successfully with specified configuration."),
+            @Step(value = "Deploys Kafka resource with configuration of Tiered Storage for Aiven plugin, pointing to Minio S3, and with image built in beforeAll.", expected = "Kafka resource is deployed successfully with Tiered Storage configuration."),
+            @Step(value = "Creates topic with enabled Tiered Storage sync with size of segments set to 10mb (this is needed to speed up the sync).", expected = "Topic is created successfully with Tiered Storage enabled and segment size of 10mb."),
+            @Step(value = "Starts continuous producer to send data to Kafka.", expected = "Continuous producer starts sending data to Kafka."),
+            @Step(value = "Wait until Minio size is not empty (contains data from Kafka).", expected = "Minio contains data from Kafka.")
+        },
```

Comment:
I think me and Luke did some changes to improve the test case with steps for retrieving the data from minio back to client once the data are gone from Kafka. Could you please add it to steps?

Reply:
Sure updated :)